src/main/java/frc/robot/subsystem/Elevator.java

```diff
@@ -0,0 +1,101 @@
+package frc.robot.subsystem;
+
+import frc.robot.Constants;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.RelativeEncoder;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+
+enum ElevatorStates {
```

Comment:
if you're gona make a seperate enum for elevator states you have to seperate it into a different file so manager can access it

Reply:
doesen't the gear change using pneumatics as well? or is it just a different motor that turns or something (you have to create a solenoid if you want different gearings and they rely on pneumatics)

Reply:
I was recommended to ignore shifting box and stick with two motors in main :) (it uses pneumatics to choose between two gears like the high or low i think). I think we're good without gearings so ima just choose low...